capstone/backend/src/main/java/com/google/servlets/Store.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.servlets;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Store {
+
+  private int totalPrice = 0;
+
+  private int storeId;
+
+  private List<Item> items = new ArrayList<Item>();
+
+  private String storeAddress;
+
+  private String storeName;
+
+  public Store(int id, String name, String address){
+    storeId = id;
+    storeName = name;
+    storeAddress = address;
+  }
+
+  public Store(StoreResult otherStore, String itemId, int priceOfItem, String itemName) {
```

Comment:
what is `StoreResult`?

Reply:
Old code mistake, store used to be called store result when I was first doing it